applicationinsights.ts

```diff
@@ -111,12 +115,13 @@ export function setup(setupString?: string) {
  * @returns {ApplicationInsights} this class
  */
 export function start() {
-    if(!!defaultClient) {
+    if (!!defaultClient) {
         _isStarted = true;
         _console.enable(_isConsole, _isConsoleLog);
         _exceptions.enable(_isExceptions);
         _performance.enable(_isPerformance);
         _heartbeat.enable(_isHeartBeat, defaultClient.config);
+        _webSnippet.enable(process.env.APPINSIGHTS_JAVASCRIPT_ENABLED == "true"); // Use environment variable by default
```

Comment:
Move environment variable & parsing it to `Config`. Use `_isSnippetInjection` (same pattern as other autocollectors)